webpage/_data/navigation.yml

```diff
@@ -7,13 +7,23 @@ main:
 
 # Site navigation links.
 site_nav:
-  - title: Video Tutorials
+  - title: Python tools
     children:
-      - title: "IOOS conda environment"
+      - title: "Installing the IOOS conda environment"
         url: https://github.com/ioos/notebooks_demos/wiki/Installing-Conda-Python-with-the-IOOS-environment
-      - title: "OWSLib"
-        url: https://geopython.github.io/OWSLib/
-      - title: "PyoosS"
-        url: https://github.com/ioos/pyoos
-      - title: "PyCSW"
-        url: http://pycsw.org/
+      - title: "Catalog driven reproducible workflows for ocean-science"
+        url: https://www.youtube.com/watch?v=05ax0lkQFrg
+      - title: "Extracting data from NOAA ERDDAP"
+        url: https://www.youtube.com/watch?v=18xZoXu1USM
+      - title: "COMT tutorial"
```

Comment:
Done in https://github.com/ioos/notebooks_demos/pull/102/commits/e82efdbc21f02f349bb7f4ef2519fe449919cebc